### PR TITLE
Adding swagger for ManagementConfiguration and ManagementAssessment o…

### PR DESCRIPTION
**Pull request by dashimi16**
Adding ManagmentConfiguration and ManagementAssocian resources. Actual resource review is on going with Gaurav. Publishing the pull request per his guidance. We should not complete this pull request before the manifest update to make the resource live.
---

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [X ] The title of the PR is clear and informative.
- [ X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [ X] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [X ] If applicable, the PR references the bug/issue that it fixes.
- [ X] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [ ]X I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [X ] My spec meets the review criteria:
  - [X ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [ X] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [X ]X [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Review by ravbhatnagar — COMMENTED** [review]
@dashimi16 As we discussed over our skype call, we need to consider a few things w.r.t exposing these APIs specially the ManagementConfiguration ones externally. So I will chat about this on our side and we can close on this hopefully by tomorrow or Monday. So I am not reviewing further and figure out the best way to proceed on this and then we can take a call.  @salameer as FYI @sarangan12 as FYI

File: specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/2015-11-01-preview/OperationsManagement.json [hunk 1]
```diff
@@ -400,6 +819,77 @@
             "description": "The container for solution.",
             "x-ms-azure-resource": true
         },
+        "ManagementAssociation": {
+            "properties": {
```
[the remark this answers is not in the document]
Comment (dashimi16) [reply specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/2015-11-01-preview/OperationsManagement.json]: It has been implemented. Missed in the swagger. will update the swagger.

File: specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/2015-11-01-preview/OperationsManagement.json [hunk 2]
```diff
@@ -400,6 +819,77 @@
...
+                "properties": {
+                    "description": "Properties for ManagementAssociation object supported by the OperationsManagement resource provider.",   
+                    "$ref": "#/definitions/ManagementAssociationProperties"                    
+                }   
+            },  
+            "description": "The container for solution.",
+            "x-ms-azure-resource": true
+        },
+         "ManagementConfiguration": {
+            "properties": {
```
[the remark this answers is not in the document]
Comment (dashimi16) [reply specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/2015-11-01-preview/OperationsManagement.json]: It has been implemented. Missed in the swagger. will update the swagger.

File: specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/2015-11-01-preview/OperationsManagement.json [hunk 3]
```diff
@@ -334,6 +681,54 @@
...
+                "applicationId": {
+                    "type": "string",
+                    "description": "The applicationId of the appliance for this Management."
+                },
+                 "parentResourceType": {
+                    "type": "string",
+                    "description": "The type of the parent resource."
+                },
+                "parameters":{
```
[the remark this answers is not in the document]
Comment (dashimi16) [reply specification/operationsmanagement/resource-manager/Microsoft.OperationsManagement/2015-11-01-preview/OperationsManagement.json]: It is not implemented and do that want to add that support.

**Review by alvadb — CHANGES_REQUESTED** [review]
Please look at example validation errors.

**Comment by ravbhatnagar** [comment]
Signing off from ARM.

**Review by alvadb — APPROVED** [review]

**Pull request merged**